tests/layout/test_table.py

```diff
@@ -2988,120 +2973,3 @@ def test_min_width_with_overflow():
 
     assert table1_td1.min_width == table2_td1.min_width
     assert table1_td1.width == table2_td1.width
-
-
-black = (0, 0, 0, 1)
-red = (1, 0, 0, 1)
-green = (0, 1, 0, 1)  # lime in CSS
-blue = (0, 0, 1, 1)
-yellow = (1, 1, 0, 1)
-black_3 = ('solid', 3, black)
-red_1 = ('solid', 1, red)
-yellow_5 = ('solid', 5, yellow)
-green_5 = ('solid', 5, green)
-dashed_blue_5 = ('dashed', 5, blue)
-
-
-@assert_no_logs
```

Comment:
Please consider preserving these test cases as unit test on [conflict resolution logic for collapsed borders](https://drafts.csswg.org/css-tables-3/#conflict-resolution-for-collapsed-borders)